### PR TITLE
Fixes the with-builder script

### PR DESCRIPTION
**Pull request by danhalson**
* Fixes an issues with the with-builder script

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Fixed
 
+- Fix local package build process (#488)
```
[the remark this answers is not in the document]
Comment (danhalson) [reply CHANGELOG.md:36]: so it only goes in the relevant `Fixed`, `Added`, or `Changed` bit when it's released?

[the remark this answers is not in the document]
Comment (sra405) [reply CHANGELOG.md:36]: Yep, so `v0.15.0` is out there and part of the release process is to rename the `unreleased` section to whatever the new version number is (https://github.com/RaspberryPiFoundation/editor-ui#release-process). Bit of a faff but that's how it is for the time being

[the remark this answers is not in the document]
Comment (danhalson) [reply CHANGELOG.md]: no worries, all done :)

**Review by sra405 — APPROVED** [review]
👍

**Review by sra405 — APPROVED** [review]
👍

**Pull request merged**